Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/TapToPlaceInspector.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor.Solvers
+{
+    /// <summary>
+    /// Custom inspector for the Tap to Place class
+    /// </summary>
+    [CustomEditor(typeof(TapToPlace))]
+    internal class TapToPlaceInspector : UnityEditor.Editor
+    {
+        protected TapToPlace instance;
+        protected SerializedProperty autoStart;
+        protected SerializedProperty rotateAccordingToSurface;
+        protected SerializedProperty keepOrientationVertical;
+        protected SerializedProperty defaultPlacementDistance;
+        protected SerializedProperty maxRaycastDistance;
+        protected SerializedProperty magneticSurfaces;
+        protected SerializedProperty debugEnabled;
+        protected SerializedProperty onPlacingStarted;
+        protected SerializedProperty onPlacingStopped;
+
+        protected virtual void OnEnable()
+        {
+            instance = (TapToPlace)target;
+            rotateAccordingToSurface = serializedObject.FindProperty("rotateAccordingToSurface");
+            defaultPlacementDistance = serializedObject.FindProperty("defaultPlacementDistance");
+            magneticSurfaces = serializedObject.FindProperty("magneticSurfaces");
+            maxRaycastDistance = serializedObject.FindProperty("maxRaycastDistance");
+            keepOrientationVertical = serializedObject.FindProperty("keepOrientationVertical");
+            autoStart = serializedObject.FindProperty("autoStart");
+            debugEnabled = serializedObject.FindProperty("debugEnabled");
+            onPlacingStarted = serializedObject.FindProperty("OnPlacingStarted");
+            onPlacingStopped = serializedObject.FindProperty("OnPlacingStopped");
+        }
+
+        public override void OnInspectorGUI()
+        {
+            RenderCustomInspector();
+        }
+
+        /// <summary>
+        /// Render the custom properties for the tap to place inspector
+        /// </summary>
+        public virtual void RenderCustomInspector()
+        {
+            serializedObject.Update();
+
+            // Disable ability to edit through the inspector if in play mode 
+            bool isPlayMode = EditorApplication.isPlaying || EditorApplication.isPaused;
+            using (new EditorGUI.DisabledScope(isPlayMode))
```

Comment:
Why do we need to disable editing this component if in play mode?

Reply:
If the user tries to toggle `AutoStart` while in play mode, the change will not get registered because the action associated with `AutoStart` is called only in Start().  I disabled it to remove the potential confusion if someone tried to toggle `AutoStart` while in play mode through the inspector only to see nothing happen.

Reply:
Not sure this is worth building an entire custom inspector for. Asking @Troy-Ferrell  for his opinion. I would vote to not make a custom inspector just t support this.

Reply:
The custom inspector has been removed.  This is what the inspector looks like now:
![image](https://user-images.githubusercontent.com/53493796/74886393-e1ca0100-532c-11ea-8491-87cb86fd3395.png)
I added a space between the inherited solver properties and the tap to place properties.